app/src/main/java/org/eyeseetea/malariacare/data/database/datasources/AuthenticationLocalDataSource.java

```diff
@@ -83,6 +82,13 @@ private void clearCredentials() {
         PreferencesState.getInstance().reloadPreferences();
     }
 
+    private void addTestOrganisation() {
```

Comment:
@manuelplazaspalacio Adding a dummy organization does not belong to the login process, it belongs to the pull process when the credentials are demo. You can move it to the pull controller where dummy organizations are added